%YAML:1.0
cameraMatrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 2.8775430269517409e+02, 0., 3.1809436997059066e+02, 0.,
       2.8704647175683243e+02, 2.3856600413872948e+02, 0., 0., 1. ]
distCoeffs: !!opencv-matrix
   rows: 8
   cols: 1
   dt: d
   data: [ 2.4456600037563642e-01, -1.6171368435162804e-02,
       -4.2838353879770470e-04, -1.4603776327410046e-04, 2e-05,
       5.6152521554734169e-01, -1.7933690356214439e-03,
       -1.8958988167269407e-03 ]
